src/Registration/Repositories/RegistrationDoctrineRepository.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Registration\Repositories;
+
+use VSV\GVQ_API\Common\Repositories\AbstractDoctrineRepository;
+use VSV\GVQ_API\Registration\Models\Registration;
+use VSV\GVQ_API\Registration\Repositories\Entities\RegistrationEntity;
+use VSV\GVQ_API\User\Repositories\Entities\UserEntity;
+
+class RegistrationDoctrineRepository extends AbstractDoctrineRepository implements RegistrationRepository
+{
+    /**
+     * @inheritdoc
+     */
+    protected function getRepositoryName(): string
+    {
+        return RegistrationEntity::class;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function save(Registration $registration): void
+    {
+        $registrationEntity = RegistrationEntity::fromRegistration($registration);
+
+        /** @var UserEntity $userEntity */
+        $userEntity = $this->entityManager->find(
```

Comment:
@stejes moeten we de user eerst ophalen? Zit die user niet gewoon mee in registration? Maar dan hadden we misschien Doctrine issue omdat die User geen managed object is.

Reply:
Ik heb even een test gedaan en precies werkt dit ook:
```
$registrationEntity = RegistrationEntity::fromRegistration($registration);

$this->entityManager->merge($registrationEntity);
$this->entityManager->flush();
```

Kan je dat eens grondig testen? Want een SQL call uitsparen is toch meegenomen en dan kunnen we dat doortrekken naar andere save methods die ook eerst een get doen.

Reply:
Ik denk dat het met enkel merge toch werkt. Er wordt in mijn testen geen user aangemaakt als de registration een onbestaande user in zich heeft. Er wordt dan wel geen foutboodschap van onszelf getoond. Ik laat de find voorlopig weg.